### PR TITLE
Fixes some things I guess

### LINKED ISSUE #1193: ORD console not updated to use newnitions or whatever its called.
Title.

### LINKED ISSUE #1269: Muni Techfab Research Broke [Atlas]
## Round ID:  N/A

 Testmerges:  N/A

## Reproduction:
Researching muni tech (macro-ballistics, etc...) doesn't give all the tech it's supposed to/missing tech after researching it.

### LINKED ISSUE #1271: Cannot rebuild Astrometrics computer
## Reproduction:
Deconstruct an Astrometrics computer. It'll give you a Station-Bounced Radio and a computer frame with an FTL Navigation board. Attempting to put the station-bounced radio into the computer frame, it will not work.

### LINKED ISSUE #1277: Pirate boarders start with suit sensors on
This is probably an oversight, since antags normally get jumpsuits that don't have suit sensors in the first place.

## Reproduction:

Be a pirate boarder, see that your suit sensors are on. (This might take multiple tries since RNG can sometimes disable your sensors)

### LINKED ISSUE #1292: Atlas general feedback
## General feedback:

- Not enough public newscasters! The only one I could find and use was in a bathroom
- Atmospherics has a single reinforced wall separating atmospherics chambers from hallways. To improve atmos security it may be possible to swap the AI chamber and atmospherics

### PR DESCRIPTION
**Pull request by Bokkiewokkie**
## About The Pull Request

It fixes the things in the changelog and the issues listed below:
fixes #1277 
fixes #1269 
fixes #1193 I think this was already fixed but the ordnance console is now not broken so yeah
fixes #1271  
fixes #1292 

## Why It's Good For The Game

Bugs are bad.
Cuts corners.
Isn't on Ansome's desk.

## Changelog
:cl:
add: Adds helmets to MT lockers.
del: Removed bomb suits and yellow caps from MT lockers.
tweak: tweaked the cost of a few munitions crates.
fix: You can actually deconstruct the astrometrics console now.
fix: IFF cards and munitions computers are now printable in the munilathe.
fix: Pirate suit sensors now always start off.
fix: Fixed the MT helmet hiding your shades with the visor up.
fix: Torpedo warhead crates now have torpedo warheads instead of missile ones.
fix: Glock ammo mags now have a lethal/nonlethal label on them.
fix: The ordnance computer now works again.
fix: Adminghosts can now interact with the armor well/pumps.
code: Actually made the recycler check if something is indestructible properly.
/:cl:

### CODE REVIEW

File: nsv13/code/modules/jobs/security/weapons.dm [hunk 1]
```diff
@@ -61,28 +61,28 @@
...
+	name = "9mm rubber Glock round"
 	id = "glock_ammo"
 	build_type = AUTOLATHE
 	materials = list(/datum/material/iron = 500)
 	build_path = /obj/item/ammo_casing/c9mm/rubber
 	category = list("initial", "Security")
 
 /datum/design/tazer
-	name = "Tazer Round"
+	name = "3mm electro-shock tazer round"
```
Comment (Karmic-Skink) [inline nsv13/code/modules/jobs/security/weapons.dm:72]: TASER.

File: nsv13/code/modules/overmap/armour/nano_pump.dm [hunk 2]
```diff
@@ -226,7 +226,7 @@
 /obj/machinery/armour_plating_nanorepair_pump/ui_act(action, params, datum/tgui/ui)
 	if(..())
 		return
-	if(!in_range(src, usr))
+	if(!(in_range(src, usr) | issiliconoradminghost(usr)))
```
Comment (Karmic-Skink) [inline nsv13/code/modules/overmap/armour/nano_pump.dm:229]: It is intended behaviour for Silicons to not be able to interact with this.

File: nsv13/code/modules/overmap/armour/nano_well.dm [hunk 3]
```diff
@@ -356,7 +356,7 @@ Starting Materials
 /obj/machinery/armour_plating_nanorepair_well/ui_act(action, params, datum/tgui/ui)
 	if(..())
 		return
-	if(!in_range(src, usr))
+	if(!(in_range(src, usr) | issiliconoradminghost(usr)))
```
Comment (Karmic-Skink) [inline nsv13/code/modules/overmap/armour/nano_well.dm:359]: It is intended behaviour for Silicons to not be able to interact with this.

File: nsv13/code/modules/jobs/security/weapons.dm (hunk 1 above)
Comment (Bokkiewokkie) [reply nsv13/code/modules/jobs/security/weapons.dm:72]: Not gonna change it for one thing if all the other mentions of it say tazer, but I might edit all of those later.

**Review by Kmc2000 — APPROVED** [review]

**Comment by jzburda** [comment]
Very nice.

**Pull request merged**